website/content/docs/projects/remote.mdx

```diff
@@ -44,10 +44,10 @@ $ waypoint up my-project/web
 
 If you have the local source checked out and your terminal working directory
 is in your project source code, you can omit the project name and use the
-`-remote` flag:
+`-local=false` option:
```

Comment:
We've also removed the old `app/project` syntax.

I'd go for something like:
~~~
Remote operations require:
- At least one [runner is installed and running](/docs/runner). Waypoint
  automatically installs a runner with `waypoint install`.
- Your project is configured with a [data source](/docs/waypoint-hcl/runner)
  so that the runners can clone the project source code.
- Your project has [remote operations enabled](/docs/waypoint-hcl/runner).

Waypoint will automatically run your operations remotely if it detects these are true, and that you
have a runner profile configured.

## Usage
### CLI


If waypoint is running your operations remotely but you'd prefer them to run locally, add the 
`-local` flag:

```shell-session
$ waypoint up -local
```
~~~